pkgs/by-name/st/stalwart-mail/webadmin.nix

```diff
@@ -14,65 +15,67 @@
   zip,
 }:
 
-rustPlatform.buildRustPackage rec {
-  pname = "webadmin";
-  version = "0.1.27";
+rustPlatform.buildRustPackage (
+  finalAttrs: with finalAttrs; {
```

Comment:
nit; I'd avoid the use of `with finalAttrs`, I think it makes it less clear what attributes are in scope and where they come from. Just use `finalAttrs.<attr>` instead, there aren't that many uses I believe (and also avoids the massive whitespace change I think)